pyproject.toml

```diff
@@ -10,6 +10,10 @@ sanic = '^19.9'
 aiomysql = '^0.0.20'
 asyncio = '^3.4.3'
 sqlalchemy = '^1.3.16'
+Sanic-Auth = '^0.2.0'
```

Comment:
Is this used anywhere?

Reply:
Not directly, but sanic_oauth will have a conniption fit if you remove it.